### ISSUE TITLE
yaml file issue

### ISSUE THREAD

**Issue by Intersect1734** [w201rdada/portfolio-Intersect1734#7]
Getting the following error after trying to build with the new w201rdada/portfolio pulled stuff from week 12 prompt.

Error in yaml::yaml.load(string, ...) : 
  Scanner error: while scanning a simple key at line 2, column 1could not find expected ':' at line 3, column 1
Calls: <Anonymous> ... parse_yaml_front_matter -> yaml_load_utf8 -> <Anonymous> -> .Call
Please delete _main.Rmd after you finish debugging the error.
Execution halted

**Comment by Intersect1734** [comment]
new files show up in the Rstudio session, but not in my Github files...
![image](https://user-images.githubusercontent.com/31350305/33303972-fac48b7e-d3ba-11e7-981c-01a52ea06d76.png)

**Comment by brooksambrose** [comment]
@Intersect1734 @rrdkent When closing an issue describe the solution for posterity!

**Comment by Intersect1734** [comment]
SOLUTION SUMMARY:
Issue seemed to be with the pull from 201rdada/portfolio parent repo. As shown in the screenshot, the files were pulling into the rstudio instance, but were not being pulled into the github directory.

Rather than pulling from the parent repo, Dan edited the output.yml directly to add TOC numbering, and I manually added the {-} tags to the headers that should not be numbered.
This captures all of the additions from the 201rdada/portfolio parent for week 12. 

Builds successfully now.

**Comment by brooksambrose** [comment]
Cool cool cool. Let me try to follow: when you say "GitHub directory" do you mean the remote? Any changes you make in the binder pod will be pushed to the remote, I hope, when you run the `_deploy.sh` script.

But the actually reason the first build failed was that something was wrong with a yaml file somewhere, as we learn from the error message. When troubleshooting in the future it is good to try to follow the error back to its source so that we can actually understand what wasn't working. But I'm glad it's been solved either way!

**Comment by Intersect1734** [comment]
Yes, that's what i meant. It didn't get there because i never got to the 'deploy' phase, i suppose. Got stuck in the 'build'.

I did see the error pointing to the yaml file. I tried re-pulling from your 201rdada/portfolio and not touching *anything* - just building right away. That still didn't work....so i don't think it was something I (intentionally) edited. 
The error refers to line 3. the _bookdown file doesn't even have 3 lines, so it must be in the output file? I couldn't figure it out.

Dan helped find a workaround... which I do recognize was a workaround ;) 
If you have any advice/idea what happened with the yaml attempt, I'd love to learn for next time.
